src/pudl/metadata/resources/ferc1.py

```diff
@@ -1694,3 +1694,145 @@
+    },
+    "tags_rate_base_category": {
+        "type": "string",
+        "description": "A category of asset or liability that RMI compiled based on  the xbrl_factoid and sometimes varies based on the .",
+        "constraints": {
+            "enum": [  # TODO 2024-05-15: need to add a null? Try w/o and see.
+                "other_plant",
+                "nuclear",
+                "transmission",
+                "net_nuclear_fuel",
+                "distribution",
+                "steam",
+                "experimental_plant",
+                "net_working_capital",
+                "general_plant",
+                "regional_transmission_and_market_operation",
+                "other_production",
+                "hydro",
+                "net_utility_plant",
+                "intangible_plant",
+                "other_deferred_debits_and_credits",
+                "net_regulatory_assets",
+                "net_ADIT",
+            ]
+        },
+    },
+    "tags_aggregatable_utility_type": {
+        "type": "string",
+        "description": (
+            "Utility type used for aggregation of the rate base table. Almost all of the values "
+            "in this column are the same as the utility_type column, but there were a few xbrl_factoid's "
+            "that are reported as utility_type total that are only applicable to a more specific utility "
+            "type."
+            # We could have chosen to update the utility_type in the transformed tables, but that would
+            # have potentially mucked up the calculations. adding this extra layer left safer and simpler
+        ),
+        "constraints": {"enum": UTILITY_TYPES_FERC1},
+    },
+    "tags_aggregatable_plant_status": {
+        "type": "string",
+        "description": (
+            "Utility plant financial status (in service, future, leased, total) used for aggregation in "
+            "the detailed and rate base tables. This column is almost always the same as the plant_status "
+            "but there were a few xbrl_factoid's that had a more general or null plant_status that we "
+            "wanted to make more specific."
+        ),
+        "constraints": {
+            "enum": ["in_service"]
+        },  # TODO 2024-05-15:  smol enum is for the rate base table only. make this a table specific enum
+    },
+    "tags_aggregatable_plant_function": {
+        "type": "string",
+        "description": (
+            "Functional role played by utility plant (steam production, nuclear production, distribution, "
+            "transmission, etc.) used for aggregation in the detailed and rate base tables. This column is "
+            "almost always the same as the plant_function but there were a few xbrl_factoid's that had a "
+            "more general or null plant_function that we wanted to make more specific. There were also "
+            "a few specific hydro plant_function's we wanted to condense into one category."
```

Comment:
having a table with the most granular itty bitty bits of what is in rate base (but no duplicate info like what is in the core accounting tables) enables us to actually have granular tags and ways to aggregate the data. My personal fav way to look at this table is by the `tags_rate_base_category`:

![image](https://github.com/catalyst-cooperative/pudl/assets/25487998/b21b9523-0d6c-4066-9f22-f7e0bcb58b54)

The three `aggregatable` columns are named  that way because there are cooresponding columns in this table of those columns. We landed on `aggregatable` because these are the columns that RMI wanted to use to aggregate the data on in their analysis.

An option for these columns is to just drop the original column and rename these guys to their core name (like this one would just be `plant_function`). They are mostly the same as the core column, but with some manual changes. I think it might be a little confusing if someone were to go back to the `core_` tables bc the value could be different.

Reply:
I think this could be resolved by going into a bit more depth in the table description above. Explain that you have three "tag" columns that are meant to be used to aggregate the data in different ways (explain what those ways are). Could also do that in the individual `tag` column descriptions -- what does it mean to aggregate the data in this way? What perspective on the data does it provide?

I'm not wild about the word "tag" in these column names, but if you feel like it has a specific function / reference to the network-x documentation we can keep it. If anything it might make more sense for it to be `aggregatable_tag` vs. `tags_aggregatable`

Reply:
@cmgosnell Is there some sign convention wonkiness going on in 1996 for `other_production`? It seems like there's also a giant lump of `<NA>` in that year in the first plot. And what's up with the discontinuity in `net_regulatory_assets` between 2016 and 2017?

Also, these plots are amazing! I've plastered them all over the socials.

Reply:
yea there is a weird nan thing in 1996. i haven't gotten a chance to debug that. and yes something dramtically change in with either the data or reporting for `net_regulatory_assets` in 2017.